docs/src/main/paradox/features/additional-directives.md

```diff
@@ -0,0 +1,72 @@
+Additional directives
```

Comment:
We need a better name here, otherwise this page either will be kitchen sink of directives, or we'll have `more-additional-directives.md`. Suggestion is "Callout box" `callout-box.md`. Who knows what a callout is? Hey it's better than "Additional directives".